rootfs/etc/systemd/system/dietpi-firstboot.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DietPi-First_boot_installation_process
+DefaultDependencies=false
+After= dietpi-preboot.service
+Before=dietpi-boot.service
```

Comment:
`network-pre.target` should be added to `Before=` as well since all those steps are meant to run before network is being initialised (like it is for PreBoot).

Reply:
Fixed with a18c393d04ea6e4e1e16024f546faaaa9545c136